src/foam/nanos/crunch/Capability.js

```diff
@@ -40,7 +41,7 @@ foam.CLASS({
     'name',
     'description',
     'version',
-    'enabled',
+    'status',
```

Comment:
think you mean 'lifecycleState' is a column here

Reply:
@zahnno yes, believe this should be lifecycleState.  
was your intent to show the current capability status - pending, granted?